.gitattributes

```diff
@@ -3,3 +3,4 @@ BookReader/plugins/*.js.map binary
 BookReader/plugins/plugin.archive_analytics.js binary
 BookReader/plugins/plugin.menu_toggle.js binary
 BookReader/plugins/plugin.tts.js binary
+BookReader/plugins/plugin.resume.js binary
```

Comment:
Let's keep this file sorted as well :+1:

Reply:
(select all the lines and `ctrl-shift-p` -> Sort Ascending in vscode :) )

Reply:
@cdrini might not be a bad idea to add a comment at the top to keep this file sorted